Src/BootCamp.Chapter/Csv/CsvRow.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+
+namespace BootCamp.Chapter.Csv
+{
+    public class CsvRow : List<string>
```

Comment:
This is a classical example of bad inheritance. Are you extending List<string> functionality or are you just using it? Is ``CsvRow`` a ``List<string>``? The answer is no. CsvRow **has** List<string>. So the relation should be composition and not inheritance. Prefer composition over inheritance.

Reply:
when I initially wrote this an CsvRow was just a List<string> and a collection of List<string>'s would have been List<List<string>> . To me this looked ugly and the solution was the CsvRow class inheriting List<string> AND Microsoft recommends to do it this way... I will add the link if I can find it again